test_pitch_shift.py

```diff
@@ -0,0 +1,28 @@
+# TODO: WRITE SOME REAL TESTS
+
+import torch
+from torch_audiomentations import Compose, PitchShift
+
+
+# Initialize augmentation callable
+apply_augmentation = Compose(
```

Comment:
It would be nice if you could wrap this code in a test function, so the test runnner doesn't automatically run this code while importing test_*.py files, looking for test functions.